src/dev/impl/DevToys/ViewModels/Tools/Text/XmlValidator/XmlValidatorToolProvider.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System.Composition;
```

Comment:
Please add `#nullable enable` at the top of the file.

Reply:
Was added.